pkg/build/builder/cmd/builder.go

```diff
@@ -45,9 +45,13 @@ func run(builderFactory factoryFunc) {
 				glog.Fatalf("Cannot parse build URL: %s", build.Spec.Source.Git.URI)
 			}
 			scmAuths := auths(sourceURL)
-			if err := setupSourceSecret(build.Spec.Source.SourceSecret.Name, scmAuths); err != nil {
+			sourceURL, err = setupSourceSecret(build.Spec.Source.SourceSecret.Name, scmAuths)
```

Comment:
It's passed in when the scm auth is created: https://github.com/jhadvig/origin/blob/BZ_1270728/pkg/build/builder/cmd/builder.go#L146-L147